### DISCUSSION TITLE
Edge/firefox/brave browser undetectable with our seleniumbase?

### DISCUSSION THREAD

**Discussion by slvamegestic** [seleniumbase/SeleniumBase#2547] [Q&A] [answered]
Would it be possible to make Edge/firefox/brave browser undetectable with our seleniumbase?

**Comment by mdmintz** [comment, marked as answer]
**UC Mode** is only for **Chrome**, **Brave**, and **Opera**.

If using Brave or Opera, set the `binary_location` in `Driver()` or `SB()`.

On a Mac, those locations are:

Brave: `binary_location="/Applications/Brave Browser.app/Contents/MacOS/Brave Browser"`
Opera: `binary_location="/Applications/Opera.app/Contents/MacOS/Opera"`

On Windows and Linux, the locations will be different. You'll need to find the location and use it for `binary_location`.

**Reply by slvamegestic** [reply]
Thanks, @mdmintz noted that. Would you mind confirming if there are no options for implementing an undetected mode on Edge browsers? While I understand this question is not related to the Selenium base, I would appreciate your input if anybody have any experience in this area

**Reply by mdmintz** [reply]
Right now, Chrome, Brave, and Opera are the only browsers officially supported by UC Mode at this time. That's already two more browsers than were originally supported when my first UC Mode video tutorial was released: <b>https://www.youtube.com/watch?v=5dMFI3e85ig</b>. There may be more browsers supported by UC Mode in the future.

If you appreciate the work, be sure to Star SeleniumBase on GitHub: ⭐ https://github.com/seleniumbase/SeleniumBase/stargazers

**Reply by bekiryen** [reply]
Which browser is best for running multiple profiles at the same time from 1 computer. Google Chrome is having issues on it..
